source/common/ssl/context_config_impl.cc

```diff
@@ -14,34 +14,35 @@
 namespace Envoy {
 namespace Ssl {
 
-namespace {
-
-Secret::TlsCertificateConfigProviderSharedPtr
-getTlsCertificateConfigProvider(const envoy::api::v2::auth::CommonTlsContext& config,
-                                Secret::SecretManager& secret_manager) {
+Secret::TlsCertificateConfigProviderSharedPtr ContextConfigImpl::getTlsCertificateConfigProvider(
```

Comment:
why did this need to be a non static private function? I don't think it uses any member variable?